test/sagemaker_tests/pytorch/training/integration/sagemaker/test_dgl.py

```diff
@@ -58,7 +58,7 @@ def test_dgl_gcn_training_gpu(ecr_image, sagemaker_regions, instance_type):
     image_cuda_version = get_cuda_version_from_tag(ecr_image)
     
     # TODO: Remove when DGL gpu test on ecs get fixed
-    if Version(image_framework_version) == Version("1.10") and image_cuda_version == "cu113":
+    if Version("1.10") <= Version(image_framework_version) < Version("1.11") and image_cuda_version == "cu113":
```

Comment:
Please change this to `Version(image_framework_version) in SpecifierSet("==1.10.*")`
```suggestion
    if Version(image_framework_version) in SpecifierSet("==1.10.*") and image_cuda_version == "cu113":
```